README.md

```diff
@@ -1,68 +1,44 @@
-# The Planetary Software Organization Technical Steering Committee
+# The Planetary Software Organization
 
-The Planetary Software Organization Technical Steering Committee (TSC) is the technical
-governing body of the Planetary Software Organization. It is described in the
-[TSC Charter][].
+The Planetary Software Organization promotes open source software in the
+planetary sciences by helping software creators and maintainers foster an
+open source community.
 
-The governing documents for the TSC are locating in the root folder of this repository.
+The Planetary Software Organization helps software creators grow
+and organize their community by providing a set of ready-to-use
+community guidelines, standards, and governance documents.  The 
+Planetary Software Organization also helps software maintainers 
+manage the growth and evolution of their open source software by 
+providing mentorship and suggestions.
 
-## TSC Scope
 
-**The TSC is responsible for those projects which are under TSC stewardship, and provides responsive input and organizational help to any projects which are not under TSC stewardship but which qualify as top level projects (TLPs).**
+## The Planetary Software Organization Technical Steering Committee
 
-The TSC exercises autonomy in managing its responsibilities.
+The Technical Steering Committee (TSC) is the governing body of the
+Planetary Software Organization, as described in its
+[Charter](TSC-Charter.md).
 
+The TSC primarily provides responsive input and organizational help
+to its Top-Level Projects and Working Groups, but is also available
+for consultation by other software projects or endeavors.
 
-### Planetary Software projects under TSC Stewardship
+Other governing documents for the TSC are locating in the root
+folder of this repository.
 
-None yet.
 
+## Planetary Software Top-Level Projects
 
-### Planetary Software projects under TSC advisement
+Learn more about Top-Level Projects.
```

Comment:
Do we want/need a Top-Level Projects document here, in symmetry to the Working-Groups document?

Reply:
Yes, but we don't have such a document to link to yet, so rather than have a dead link here, I just left this as a plain sentence that we can link when we have the document.